packages/lesswrong/lib/collections/comments/collection.ts

```diff
@@ -92,7 +92,12 @@ makeEditable({
       if (comment.parentCommentId) { return {id: ('parent:' + comment.parentCommentId), verify: false}}
       return {id: ('post:' + comment.postId), verify: false}
     },
-    order: 25
+    order: 25,
+    // Todo(PR): So far we've been pretending that mentions is just another type of pingback, which is true enough
+    //  pingbacks are rn enabled only in a few places though, so to make the mentions work, we'd need to enable them
+    //  in more places. At the very least here, maybe in other places. 
+    //  Leaving this in as PR discussion
+    pingbacks: true, 
```

Comment:
I think having mentions in comments and posts covers the use cases that matter most. I think we want them in tag _discussions_, but not tags themselves, and I think tag-discussions just use comments?

(We may not have pingbacks enabled in comments, in which case I definitely agree we'd want to turn that on and/or figure out what to do instead)

Reply:
this line enables the pingbacks in the comments. and indeed the tag-discussions are just comments - I've tested doing mentions on those.

Reply:
To confirm and resolve this one: for now we are ok with mentions working only on posts and comments?

Reply:
Yes.